test/e2e/cypress-ui/cypress/support/constants.js

```diff
@@ -16,8 +16,8 @@ export const clusterpools_path = "/clusterpools"
 export const clusterclaims_path = "/clusterclaims"
 
 // API URL
-export const apiUrl = Cypress.config().baseUrl.replace("multicloud-console.apps", "api") + ":6443";
-export const ocpUrl = Cypress.config().baseUrl.replace("multicloud-console.apps", "console-openshift-console.apps");
-export const prometheusUrl = Cypress.config().baseUrl.replace("multicloud-console.apps", "prometheus-k8s-openshift-monitoring.apps");
+export const apiUrl = Cypress.config().baseUrl;
```

Comment:
I would think this changes would be breaking stuff because the ACM console URL != api ?URL != OCP UI URL

Reply:
Yes, I thought so too, but looks like the sample acmLogin test has passed. Either way that is a sample test which will likely go away soon. I also checked with David and he said this change shouldn't affect that test.

Reply:
The acmLogin test uses cy.visit() which only uses Cypress.config().baseUrl, so this shouldn't cause any breaking change

Reply:
So I think this cypress was copied from ACM cypress which is assuming the baseUrl passed in is the ACM UI URL.  In our case, we are passing in the OCP UI URL, so that is why we have the difference in logic.

Reply:
Yes, I took this from the other repo I'm working as a skeleton to hook up into Prow but I guess I didn't remove all the flesh... I'll spend the next few days cleaning it up so we have less of these.